rules/k8s_audit_rules.yaml

```diff
@@ -521,7 +521,7 @@
 
 - rule: K8s Serviceaccount Created
   desc: Detect any attempt to create a service account
-  condition: (kactivity and kcreate and serviceaccount and response_successful)
+  condition: (kactivity and kcreate and serviceaccount and non_system_user and response_successful)
```

Comment:
I think the rule is for auditing purpose. It should record all the sa that created.just the other rule `K8s Serviceaccount Deleted`

Reply:
I agree--the rule is intended to be a generic sensor, generating an alert any time a service account is created. For that reason, I think it should also generate alerts for service accounts created by system accounts.